plasmapy/classes/sources/tests/test_openpmd_hdf5.py

```diff
@@ -16,7 +16,7 @@ class TestOpenPMD2D:
     h5 = openpmd_hdf5.HDF5Reader(hdf5=os.path.join(rootdir, "data00000255.h5"))
 
     def test_has_electric_field_with_units(self):
-        assert self.h5.electric_field.to(u.V / u.m)
+        assert isinstance(self.h5.electric_field.to(u.V / u.m), u.Quantity)
```

Comment:
So the inner statement checks whether `electric_field` converts to V/m, and then `isinstance` returns a boolean on the type check. Neat solution!